modules/videoio/src/cap_avfoundation_mac.mm

```diff
@@ -293,7 +293,7 @@ - (IplImage*)getOutput;
 
     [mCaptureSession release];
     [mCaptureDeviceInput release];
-    [mCaptureDevice release];
+    // [mCaptureDevice release]; fix #7833
```

Comment:
Hi @alalek 

Sorry I did not know that this was already fixed as no traces on the issue. Please let know what I need to do? Just need this issue fixed as it is pretty annoying.

Regards.

Alex